src/state/routing/types.ts

```diff
@@ -87,6 +87,13 @@ export interface QuoteResult {
 export const INITIALIZED = 'Initialized'
 export const NO_ROUTE = 'No Route'
 
-export type GetQuoteResult = QuoteResult | typeof INITIALIZED | typeof NO_ROUTE
+export type GetQuoteError = typeof INITIALIZED | typeof NO_ROUTE
+export type GetQuoteResult = QuoteResult | GetQuoteError
```

Comment:
Shouldn't `GetQuoteResult` now be `TradeResult | GetQuoteError`? `QuoteResult` is no longer directly returned, and this affects the typing of the rtk-query slice. It is only named GetQuoteResult because it is returned by GetQuoteQuery, so if the query returns something else (TradeResultQuote), then things should be renamed.

Reply:
thoughts on the name `getSwapTradeResult` for the queryFn/hook name?